### PR TITLE
Update Android registration code snipped

### PR DESCRIPTION
**Pull request by danielpassos**
See: [AGPUSH-1360](https://issues.jboss.org/browse/AGPUSH-1360)

### CODE REVIEW

**Comment by danielpassos** [comment]
@matzew @sebastienblanc @secondsun Could you give it a spin?

File: admin-ui/app/views/example.html [hunk 1]
```diff
@@ -108,37 +106,34 @@
...
+                    .setSecret(SECRET)
+                    .asRegistrar();
 
-            registration.register(getApplicationContext(), new Callback() {
-                private static final long serialVersionUID = 1L;
+            PushRegistrar registrar = RegistrarManager.getRegistrar("register");
+            registrar.register(getApplicationContext(), new Callback<Void>() {
+                @Override
+                public void onSuccess(Void data) {
+                    Log.i("PushApplication", "Registration Succeeded!");
```
Comment (secondsun) [inline admin-ui/app/views/example.html]: We usually use a constant field "TAG" for log tagging.

Comment (danielpassos) [inline admin-ui/app/views/example.html]: Done!

**Comment by matzew** [comment]
@danielpassos @sebastienblanc should it say, somewhere, somehow which version of the API it is? 2.x for instance?

**Comment by danielpassos** [comment]
Yes, why not?!

**Comment by matzew** [comment]
@danielpassos ok, let's do it! @sebastienblanc any comments?

**Comment by sebastienblanc** [comment]
Let's put the version in a code comment at the beginning of the snippet , wdyt ?

**Comment by matzew** [comment]
ok

On Tue, Apr 14, 2015 at 11:05 AM, Sebastien Blanc <notifications@github.com>
wrote:

> Let's put the version in a code comment at the beginning of the snippet ,
> wdyt ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/524#issuecomment-92707588>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Comment by danielpassos** [comment]
_AeroGear Android >=2.0 code to register Android device in UPS_ wdyt?

**Comment by secondsun** [comment]
@danielpassos If you put it in the javadoc for the snippit why not @link it to aerogear-android-push?

**Comment by matzew** [comment]
merged...

**Pull request merged**